src/index.css

```diff
@@ -19,4 +23,55 @@ h4 {
     flex-wrap: wrap;
     align-items: flex-start;
     justify-content: center; 
+}
+
+.carousel {
+    box-shadow: 0 6px 8px rgb(94, 94, 94);
+}
+
+.youtube-thumb {
+    box-shadow: 0 0 14px rgb(62, 62, 62); 
```

Comment:
Curious why the different box shadow configs? I don't think it looks bad- just wondering

Reply:
Tried uniform config but did not look good for both elements---if using box-shadow subtle is better imo